packages/components/src/Text/Text.tsx

```diff
@@ -10,7 +10,7 @@ type Props = {
    */
   as?: TextElement;
   children: TypographyProps<TextElement>["children"];
-  className: TypographyProps<TextElement>["className"];
```

Comment:
Was hitting a type error because `className` was a required prop. I don't think we actually want it to be required?

Reply:
agree 👍